README.md

```diff
@@ -6,29 +6,24 @@ Provides standard objects and functions for working with dates and times.
 
 ## Champions
 
--   Philipp Dunkel ([@pipobscure](https://github.com/pipobscure))
--   Maggie Johnson-Pint ([@maggiepint](https://github.com/maggiepint))
--   Matt Johnson-Pint ([@mattjohnsonpint](https://github.com/mattjohnsonpint))
--   Brian Terlson ([@bterlson](https://github.com/bterlson))
--   Shane Carr ([@sffc](https://github.com/sffc))
--   Ujjwal Sharma ([@ryzokuken](https://github.com/ryzokuken))
--   Philip Chimento ([@ptomato](https://github.com/ptomato))
--   Jason Williams ([@jasonwilliams](https://github.com/jasonwilliams))
--   Justin Grant ([@justingrant](https://github.com/justingrant))
+- Philipp Dunkel ([@pipobscure](https://github.com/pipobscure))
+- Maggie Johnson-Pint ([@maggiepint](https://github.com/maggiepint))
+- Matt Johnson-Pint ([@mattjohnsonpint](https://github.com/mattjohnsonpint))
+- Brian Terlson ([@bterlson](https://github.com/bterlson))
+- Shane Carr ([@sffc](https://github.com/sffc))
+- Ujjwal Sharma ([@ryzokuken](https://github.com/ryzokuken))
+- Philip Chimento ([@ptomato](https://github.com/ptomato))
+- Jason Williams ([@jasonwilliams](https://github.com/jasonwilliams))
+- Justin Grant ([@justingrant](https://github.com/justingrant))
 
 ## Status
 
-This proposal is currently [Stage 2](https://github.com/tc39/proposals#stage-2).
+This proposal is currently [Stage 3](https://github.com/tc39/proposals#stage-3).
+Stage 3 reviewers were: Richard Gibson, Bradley Farias, and Daniel Ehrenberg.
 
-At the time of writing, all the major design decisions that we are aware of have been made.
-The proposal is under review by the TC39 delegates and the bar for making subsequent changes is high.
-Nonetheless, changes may occur as the result of feedback from TC39 reviews and implementation in JS engines.
-
-Stage 3 Reviewers:
-
--   Richard Gibson
--   Bradley Farias
--   Daniel Ehrenberg
+This proposal is now in the hands of ECMAScript engine implementers, so the bar for making API changes is extremely high.
```

Comment:
Should we also mention that we do not expect unflagged implementations to ship yet?

Reply:
Yes, absolutely.

Reply:
Great suggestion!  I drafted the text below and will include it in the final version of this PR. Holler if there are concerns. 

> **NOTE: Although this proposal's API is not expected to change, implementers of this proposal MUST NOT ship unflagged Temporal implementations until IETF standardizes timezone/calendar string serialization formats. See #1450 for updates.**

I'm also going to reorganize the proposal README so the flag warning and the "don't use the polyfill in prod" warnings are more prominent, just to be sure.